WcaOnRails/app/views/shared/_user_avatar.html.erb

```diff
@@ -1,5 +1,6 @@
 <% avatar_symbol = defined?(show_pending) && show_pending ? :pending_avatar : :avatar %>
   <% url = user.send(avatar_symbol).url(:thumb) + ( defined?(break_cache) ? "?#{Time.now.to_f}" : "" ) %>
+  <% url.gsub!("localhost:3000", "worldcubeassociation.org") %>
```

Comment:
Hmm, I believe we don't need that, the origin just depends on environment. Why was that change needed?

Reply:
@jonatanklosko Thank you for catching this, this is workaround I've been using to let the images display locally so I could get a better idea of how everything looked. I had been reverting this before my commits but it looks like I forgot this time.